cl/scrapers/tasks.py

```diff
@@ -374,24 +154,19 @@ def extract_recap_pdf(pks, skip_ocr=False, check_if_needed=True):
             # hasn't disabled early abortion.
             processed.append(pk)
             continue
+
```

Comment:
It looks like you tweaked the logic here so that OCR_UNNECESSARY is never set, and so that skip_ocr is never checked. Can you please review?